### ISSUE TITLE
TypeError: Typed property DarkGhostHunter\Preloader\Opcache::$status must be array, bool used 

### ISSUE THREAD

**Issue by aksafan** [DarkGhostHunter/Preloader#40]
Hello @DarkGhostHunter 
I have troubles with using `getList()` method on my develop server:
```
TypeError: Typed property DarkGhostHunter\Preloader\Opcache::$status must be array, bool used in /projectName/vendor/darkghosthunter/preloader/src/Opcache.php:29
Stack trace:
#0 /projectName/vendor/darkghosthunter/preloader/src/Opcache.php(49): DarkGhostHunter\Preloader\Opcache-&gt;getStatus()
#1 /projectName/vendor/darkghosthunter/preloader/src/Preloader.php(87): DarkGhostHunter\Preloader\Opcache-&gt;getScripts()
#2 /projectName/vendor/darkghosthunter/preloader/src/Preloader.php(77): DarkGhostHunter\Preloader\Preloader-&gt;prepareLister()
#3 /projectName/PreloadGenerator.php(12): DarkGhostHunter\Preloader\Preloader-&gt;getList()
```

This happens when opcache is disabled and `opcache_get_status()` returns false (bool) instead of an array. That's why i have an exception here:
```
    public function getScripts() : array
    {
        return $this->getStatus()['scripts'];
    }
```

My suggestion is to check wether it bool (false) or array and throw more intuitive exception when it is false.
Made a PR with this changes - https://github.com/DarkGhostHunter/Preloader/pull/39

**Comment by DarkGhostHunter** [comment]
Read the PR, should fix the problem that enabling Preloader on non-opcache-enabled environments (it should throw the error).

Still, the package is not meant to be activated if Opcache is disabled, wherever it is.

**Comment by aksafan** [comment]
Thank you for answer @DarkGhostHunter,

My point is to clarify the exception message without changing the main behavior. There still will be an exception when opcache is disabled, cause `opcache_get_status()` will return false. But this exception will be clear to end user, cause it took me some time to understand what is going on when i got `TypeError`.

**Comment by DarkGhostHunter** [comment]
Fixed in v2.0.4.
